rockcraft/services/image.py

```diff
@@ -48,8 +48,9 @@ def __init__(
         project: models.Project,
         work_dir: Path,
         build_for: str,
+        **kwargs,
     ):
-        super().__init__(app, services, project=project)
+        super().__init__(app, services, project=project, **kwargs)
```

Comment:
I thought it was very "bureaucratic" to have to add the new parameter to all services, even those that don't use it (like this ImageService)